### ISSUE TITLE
[Feature request] The nodes interface returns the version of the node

### ISSUE THREAD

**Issue by zhangyongding** [rqlite/rqlite#2001]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
I hope to quickly know the versions of all nodes in the cluster through the nodes interface

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by otoolep** [comment]
Reasonable request, would be convenient and not too much work.

But I want to check that do you know that the software version is returned by every HTTP request made to a node. So you can do this now by sending, say, a `/status` request to each node in your cluster and checking the version that each node is running that way.

```bash
$ curl -sI http://localhost:4001/status | grep 'X-Rqlite-Version' 
X-Rqlite-Version: v8.36.0
$ curl -sI http://localhost:4001/status | grep 'X-Rqlite-Version' | cut -d ' ' -f2
v8.36.0
$ 
```

**Comment by otoolep** [comment]
Available in v8.36.3
